addOns/scripts/scripts.gradle.kts

```diff
@@ -17,8 +17,9 @@ zapAddOn {
                 dependencies {
                     addOns {
                         register("automation") {
-                            version.set(">=0.24.0")
+                            version.set(">=0.31.0")
                         }
+                        register("commonlib")
```

Comment:
Noticed it after merging... Does this need a version to be set explicitly?

Reply:
Not in this case, the `automation` already imposes a/the minimum.